### ISSUE TITLE
RFE: portablectl should have a `INSTALL_CLIENT_SIDE` feature ala systemctl

### ISSUE THREAD

**Issue by jaymzh** [systemd/systemd#10237]
`systemctl` will obey `$SYSTEMCTL_INSTALL_CLIENT_SIDE` option for when it's being called from a generator or other time when it shouldn't ask systemd to do work and instead do it itself.

We'd like to be able to use `portablectl` inside of generators and thus a similar feature for it would be very useful.

**Comment by boucman** [comment]
Yes, that would be extremely usefull for the embedded world where everything is generated offline
